### PR TITLE
#1593: Resolve Assorted Python Dependency Issues

### PR DESCRIPTION
**Pull request by tst-cjeffries**
Updates to fix various python dependency issues:

- Portable shebang fixed and documentation changed.
- Pyyaml now installs to the correct location.
- OpenGEE rpms now require python27-lxml (to ensure that it gets installed automatically).

To test:

- Build portable on el6 and verify that it builds with and works with python 2.7.
- Build and install on el6 and verify that glc_assembler works.
- Take built RPMs and install on a different el6 machine. Verify that cutter works.

These changes were made in conjunction with documentation changes on the [wiki](https://github.com/google/earthenterprise/wiki/Portable-Server#run-time-prerequisites).

### CODE REVIEW

File: earth_enterprise/src/portableserver/build.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/python
+#! /usr/bin/env python2.7
```
Comment (tst-dsugar) [inline earth_enterprise/src/portableserver/build.py]: This probably makes sense to be fully consistent between portable and everything else now....

**Comment by tst-dsugar** [comment]
Should this target 5.3.3?

File: earth_enterprise/src/portableserver/build.py (hunk 1 above)
Comment (tst-cjeffries) [reply earth_enterprise/src/portableserver/build.py]: Yes, plus it was causing portable to be using python 2.6.

**Comment by tst-cjeffries** [comment]
Originally I had it for 5.3.3, but Rick wanted these changes applied to 5.3.2.1, and then merged to 5.3.3. It makes sense to do it that way because the problems here were introduced with the python dependency changes so they need to be in 5.3.2.1.

**Comment by tst-jzmolek** [comment]
Reviewed and tested. Looks good to me. Although shouldn't this target release_5.3.3?

**Pull request merged**